pkg/network/event_windows.go

```diff
@@ -116,6 +117,7 @@ func FlowToConnStat(cs *ConnectionStats, flow *C.struct__perFlowData, enableMono
 	} else {
 		// V6 Address
 		srcAddr, dstAddr = convertV6Addr(flow.localAddress), convertV6Addr(flow.remoteAddress)
+		log.Infof("Converted v6 address %v %v", srcAddr, dstAddr)
```

Comment:
Is this for debugging? I don't think we want this logging for every IPv6 address.

Reply:
See this PR is WIP. Ignore.